modules/utils.js

```diff
@@ -10,8 +10,32 @@
  * governing permissions and limitations under the License.
  */
 
+function cleanURL(url) {
+  const u = new URL(url);
+  u.search = '';
+  u.hash = '';
+  return u.toString();
+}
 export const urlSanitizers = {
-  full: (url = window.location.href) => new URL(url).toString(),
+  /**
+   * Returns the sanitized url: the origin and the path (no query params or hash)
+   * If no url is provided, it defaults to window.location.href.
+   * @param {string} url (default: window.location.href) The url to sanitize
+   * @returns {string} The sanitized url
+   */
+  full: (url = window.location.href) => cleanURL(url),
```

Comment:
```suggestion
  full: (url = window.location.href) => urlSanitizers.path(url),
```
If they are the same, one can call the other...

Reply:
That won't go through the linter, because you cannot access `urlSanitizers` before it has been defined.